src/platform/P6/args.gni

```diff
@@ -1,23 +1,23 @@
-# Copyright (c) 2021 Project CHIP Authors
+#Copyright(c) 2021 Project CHIP Authors
```

Comment:
nit: It looks like a bunch of spacing got all weird

Reply:
Fixed up